src/scripts/domParsers/domUtils.ts

```diff
@@ -217,27 +217,27 @@ export class DomUtils {
 		let tagsToTurnIntoDiv = [DomUtils.tags.main, DomUtils.tags.article, DomUtils.tags.figure, DomUtils.tags.header, DomUtils.tags.center];
 
 		// ... and for everything else, we replace them with an equivalent, preserving the inner HTML
-		DomUtils.domReplacer(doc, tagsToTurnIntoDiv.join(), (node: HTMLElement) => {
+		DomUtils.domReplacer(doc, tagsToTurnIntoDiv.join(), async (node: HTMLElement) => {
 			let div = document.createElement("div");
```

Comment:
Does document.createElement work here and not need to be moved to offscreen document?

Reply:
I see that this is a part of the content scripts, i.e. `clipper.js` and not the service worker, i.e. `chromeExtension.js` and should hence continue working as expected.